packages/toolkit/src/createAction.ts

```diff
@@ -275,7 +277,7 @@ export function createAction(type: string, prepareAction?: Function): any {
     return { type, payload: args[0] }
   }
 
-  actionCreator.toString = () => `${type}`
+  actionCreator.toString = () => type
```

Comment:
i think this should be left as is - we don't do any runtime protection against non-strings so we need to make sure this returns a string regardless